ssl_setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
I would move it under the demo if needed (or is it for manual testing only?).
But demo could use non-authenticated mode for simplicity, up to you

Reply:
@oleg-nenashev This file is needed to setup SSL on Kafka. I will add comments to this file for clarification.

Reply:
works for me